### ISSUE TITLE
Add URL of license to top of license file

### ISSUE THREAD

**Issue by FranklinYu** [chocolatey-community/chocolatey-coreteampackages#1294]
As we all know it is strongly recommended to put license URL on top of license file. A positive example is [Torrent File Editor](https://chocolatey.org/packages/torrent-file-editor). The core team packages are often used as example and role-model for new users; it would be misleading if most core team packages don’t follow best practice.

I hereby propose we add the URL to top of all licenses in this repository. I can work for the PR but I would like to have consensus first.

**Comment by majkinetor** [comment]
My opinion is that its better like this:

* I don't like editing 3rd party files. We take license as is from upstream location. There is also clausule in many licenses that you can redistribute software as long as you don't modify any files. 
* There is already a location in file VERIFICATION.txt on the last line for each embedded package.

**Comment by AdmiringWorm** [comment]
I agree with @majkinetor here. The license files should be equivalent to the ones found upstream, and IMO one of the reason for the `VERIFICATION.txt` file is to verify the included files, as such the url for the license should be defined there.

> As we all know it is strongly recommended to put license URL on top of license file.

First time I have ever heard of it being strongly recommended, sure it is part of `choco new`, but as every other file created by that command it is just a template. A suggestion if you will.

Having the license url in the `VERIFICATION.txt` file is also something we have been doing pretty much ever since we started embedding packages in this repo (around two years ago, I believe).

**Comment by mkevenaar** [comment]
If you look at the license template when you run `choco new` the URL is actually on top:

https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure.app/templates/ChocolateyLicenseFileTemplate.cs

As core team I think we should set the example and update our LICENSE.txt files.

From my personal point of view I do agree with both of you.

**Comment by pascalberger** [comment]
I'm with @majkinetor and @AdmiringWorm on this one, that upstream files should not be changed (and in some cases it's also legally not allowed)

**Comment by FranklinYu** [comment]
I do agree with @majkinetor and @AdmiringWorm but the reality is that we have two parts of the system that don't agree with each other. Either we update the packages, or we update the template to move the URL from `LICENSE.txt` to `VERIFICATION.txt`, in order to stop confusing maintainers.

**Comment by FranklinYu** [comment]
I created chocolatey/choco#1950

Thank everyone!
